functions-and-scope.js

```diff
@@ -78,6 +120,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Zorg ervoor dat jouw functie ook werkt als we een andere array willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
 
+function getHighestGrade(gradesArray) {
+            if (!Array.isArray(gradesArray) || gradesArray.length === 0) {
+            return console.log(`Input is empty or not valid`);
```

Comment:
Mooie toevoeging, maar error afhandeling is natuurlijk nog niet echt nodig voor deze opdracht :P